public/css/list/item-list.less

```diff
@@ -43,74 +43,20 @@
 
 // Layout
 
-.item-list .list-item {
-  &.load-more a {
+.item-list .load-more {
+  display: flex;
+
+  a {
     flex: 1;
     margin: 1.5em 0;
     padding: .5em 0;
   }
 }
 
-.item-list.minimal {
+.item-list.minimal-item-layout {
   > .empty-state {
     padding: .25em;
   }
```

Comment:
this is an obsolete rule, it's am empty-state-bar nowadays.

Reply:
Removed